client/signup/steps/plans/index.jsx

```diff
@@ -91,7 +92,8 @@ module.exports = React.createClass( {
 					href={ this.comparePlansUrl() }
 					className='plans-step__compare-plans-link'
 					onClick={ this.handleComparePlansLinkClick.bind( null, 'footer' ) }>
-					{ this.translate( 'Compare Plans' ) }
+						<Gridicon icon="clipboard" size="18" />
```

Comment:
For good measure, the size prop should be passed as a number not a string: `size={ 18 }`